src/main/java/gb/bourne2code/warframe/opencv/controllers/FileSelectController.java

```diff
@@ -3,21 +3,26 @@
 import gb.bourne2code.warframe.opencv.Manager;
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
+import javafx.scene.control.ChoiceBox;
 import javafx.scene.control.TextField;
-import javafx.scene.image.ImageView;
 import javafx.stage.FileChooser;
 
+import javax.imageio.ImageIO;
 import java.io.File;
+import java.io.IOException;
 
 /** Controls the login screen */
 public class FileSelectController {
-        @FXML private TextField fileLocation;
-        private Manager manager;
+    @FXML public ChoiceBox platformSelect;
+    @FXML private TextField fileLocation;
+    @FXML private Manager manager;
 
     public void initManager(final Manager manager) {
         this.manager = manager;
     }
 
+
+
```

Comment:
extra spaces? maybe javadoc instead?

Reply:
that was accidental, i think we should add javadocs for all other files (not added by this pull request) in another pr